ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -147,20 +148,20 @@ public SqlStruct value() {
    * @param fieldName the field name, where any alias is ignored.
    * @return the value field if found, else {@code Optiona.empty()}.
    */
-  public Optional<Field> findValueField(final String fieldName) {
-    return doFindField(fieldName, value.fields());
+  public Optional<Column> findValueField(final String fieldName) {
+    return doFindColumn(fieldName, value);
   }
 
   /**
-   * Find the index of the field with the supplied exact {@code fullFieldName}.
+   * Find the index of the field with the supplied exact {@code fullColumnName}.
    *
-   * @param fullFieldName the exact name of the field to get the index of.
+   * @param fullColumnName the exact name of the field to get the index of.
    * @return the index if it exists or else {@code empty()}.
    */
-  public OptionalInt valueFieldIndex(final String fullFieldName) {
+  public OptionalInt valueFieldIndex(final String fullColumnName) {
```

Comment:
```suggestion
  public OptionalInt valueColumnIndex(final String fullColumnName) {
```

I won't continue with calling out every use of `field` - you get the general idea ;) . Search for field in the class...

Reply:
😂